docs/api_reference/postman_colletion.md

```diff
@@ -0,0 +1,712 @@
+---
+sidebar_position: 6
+---
+
+# Postman Collection
+
+[Postman is a desktop application](https://www.postman.com/) that allows you to configure and intereact with HTTP-based APIs. Callbell REST API implements several endpoints that you can interact with. 
+
+To utilize our Postman collection you need to:
+
+1. copy the raw json file below:
+
+<details>
+<summary> Postman Collection raw JSON</summary>
+
+```json
```

Comment:
Cool 😎 Do you think it would be possible to just add a markdown link here and add the actual JSON under public? This way people should be able to download the file by clicking (I'm not 100% sure if this works but worth a try!)

Reply:
I also think is doable that way, worth trying 👍🏼